lessons/Algebra2/exploring-quadratic-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,214 @@
+
+A quadratic relationship often means that one or more variables is _interacting_ with another variable. In our miles-per-gallon example, it turns out that a linear increase in speed has a _more than linear effect_ on mpg!
+|===
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely quadratic?
+
+=== Investigate
+
+You already know that linear relationships can be described with terms like "slope" and "y-intercept". Parabolas have their own terms:
+
+[cols=".^3a,^.^1a", stripes="none", frame="none"]
+|===
+|
+- @vocab{vertex} - In a quadratic relationship, the vertex is the point at which a parabola "changes direction", and goes from climbing to sinking (or vice versa).
+- @vocab{maxima} - The highest point in a parabola that "opens down"
+- @vocab{minima} - The lowest point in a parabola that "opens up"
+| @image{images/opens.png, 175}
+
+|
+- @vocab{y-intercept} - Like linear models, parabolas always cross the y-axis once when x=0.
+|  @image{images/y-intercept.png, 175}
+
+|
+- @vocab{x-intercepts} / @vocab{roots} - All linear functions cross the x-axis once, when y=0. These are sometimes called "roots", or "zeros" of the function. A quadratic function can cross the x-axis once, twice, or even __no times at all!__
+|
+@image{images/roots.png, 200}
+|===
+
+=== Synthesize
+
+- How are quadratic models _similar_ to linear ones?
+- How are they _different?_
+
+== Fitting Quadratic Models @duration{25 minutes}
+
+=== Overview
+
+Students are introduced to quadratic models, and extend the model-fitting techniques from the @lesson-link{exploring-linear-models} lesson into quadratic relationships.
+
+=== Launch
+
+You should already be familiar with the different forms of linear models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Slope-intercept		| Point-slope				| Standard 	
+| @math{y = mx+b}		| @math{y-y_1 = m(x-x_1)}	| @math{Ax+By = C}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+The same is true for quadratic models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Standard Form 		| Vertex Form			| Factored Form
+| @math{y=ax^2+bx+c}	| @math{y=a(x−h)^2+k}	| @math{y=a(x−r_1)(x−r_2)}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+- Standard Form makes it easy to find the @vocab{y-intercept} of the parabola @math{c}
+- Vertex Form makes it easy to find the @vocab{vertex} (@vocab{minima} or @vocab{maxima}) of the parabola
+- Factored Form makes it easy to find the @vocab{roots} of the parabola
+
+[.lesson-instruction]
+Given what we've already found out about this dataset, which form should we start with?
+
+In this case, we've already come to some conclusions about the @vocab{vertex} of our parabola, where cars are most fuel-efficient: *45mph*. That makes Vertex Form a useful place to start, since we already know the value of @math{H}:
```

Comment:
While I might place the axis of symmetry at 45, I wouldn't choose it as the most fuel efficient speed by way of the question asked in the previous sections.

Reply:
Addressed, though this is dependent on whether you approve of my changes to address [your earlier comment](https://github.com/bootstrapworld/curriculum/pull/1385#discussion_r1226903974).